apps/tlv-app/grails-app/assets/javascripts/time-lapse.js

```diff
@@ -491,10 +491,33 @@ function updatePqe( pixel ) {
     } );
 }
 
+// Updates the security banner to match the current image security-classification
+function updateSecurityBanner() {
+	$('#securityBanner').removeClass("unclassified");
+	$('#securityBanner').removeClass("secret");
+	$('#securityBanner').removeClass("top-secret");
+	switch(tlv.layers[ tlv.currentLayer ].metadata.security_classification) {
+		case "UNCLASSIFIED":
+			$('#securityBanner').text("UNCLASSIFIED");
+			$('#securityBanner').addClass("unclassified");
+			break;
+		case "SECRET":
+		  	$('#securityBanner').text("SECRET");
+		  	$('#securityBanner').addClass("secret");
+		  	break;
+		case "TOP-SECRET":
+			$('#securityBanner').text("TOP-SECRET");
+			$('#securityBanner').addClass("top-secret");
+			break;
+		default:
```

Comment:
Add default to catch fall through errors with default flag for dynamic classification systems.

Reply:
Yes, we need to figure out how to have a default classification based on the omar-security config. However, the style should be good since it is set initially and then removing all the classes at the beginning of the function should revert it back to whatever the default style is.